src/scaffold/template/maven/gen/maven/pom.xml

```diff
@@ -10,6 +10,12 @@
     <packaging>jar</packaging>
     <description>{{ customConfig.title }}</description>
 
+    <properties>
+        <java.version>1.8</java.version>
```

Comment:
Hmm. I only remind those properties to set the java version:
```
<maven.compiler.target>1.8</maven.compiler.target>
<maven.compiler.source>1.8</maven.compiler.source>
```
See: https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html

What is the property `java.version` for?

Reply:
You are right, <java.version> is used in spring boot projects to define the source and target java version. But in this case we should obviously use the <maven.compiler.target> and <maven.compiler.source> properties.